server/src/test/java/com/microsoft/java/bs/core/internal/server/BuildTargetServerIntegrationTest.java

```diff
@@ -477,6 +535,87 @@ void testCompilingSingleProjectServer() {
     });
   }
 
+  @Test
+  void testIncompatibleUserJavaHomeProjectServer() {
```

Comment:
Makes more sense to put it into `LifecycleServiceIntegrationTests`?

Reply:
Are you proposing to create a new test file? I didn't find any existing test file called `LifecycleServiceIntegrationTests`.

Reply:
Yes it is.

Reply:
Fixed in commit https://github.com/microsoft/build-server-for-gradle/pull/165/commits/8ed01255f35de1f7ab772357cfc30e44a3f26174